docs/092-ht_change_voice.md

```diff
@@ -1,3 +1,47 @@
 # How To Change Your Assistant's Voice
 
-**WIP**
+## Using TTS server plugin
+
+Open the file for editing.  `nano ~/.config/mycroft/mycroft.conf`.
+
+If your file is empty, or does not have a `"tts"` section, you need to create it.  Add this to your config
+
+```json
+{
+    "tts": {
+        "ovos-tts-plugin-server": {
+            "voice": "amy-low"
+        }
+    }
+}
+```
+
+Execute `systemctl --user restart ovos` to reload configuration.
+
+For avaliable voices please refer to [piper-voices section](#voices-avaliable-for-piper-plugin-)
+
+## Using local piper plugin
+
+Open the file for editing.  `nano ~/.config/mycroft/mycroft.conf`.
+
+If your file is empty, or does not have a `"tts"` section, you need to create it.  Add this to your config
+
+```json
+{
+    "tts": {
+        "module": "ovos-tts-plugin-piper",
+        "ovos-tts-plugin-piper": {
+            "voice": "alba-medium"
+        }
+    }
+}
+```
+
+Execute `systemctl --user restart ovos` to reload configuration.
+
+For avaliable voices please refer to [piper-voices section](#voices-avaliable-for-piper-plugin-)
+
+# Voices avaliable for [piper-plugin](https://github.com/OpenVoiceOS/ovos-tts-plugin-piper/) <a id="Voices"></a>
```

Comment:
Why the html tag?

Reply:
Sorry, was trying to make anchors work, wasn't sure how github does it. Thats already removed